apps/mobile/src/packages/users/users.ts

```diff
@@ -15,6 +15,7 @@ export { UserValidationMessage } from "./libs/enums/enums";
 export {
 	type UserDto,
 	type UserGetAllResponseDto,
```

Comment:
Can we completely remove UserGetAllResponseDto or is it still used somewhere?

Reply:
If only in Backend, then you can remove from mobile?

Reply:
Done